src/sys/signal.rs

```diff
@@ -210,6 +211,19 @@ libc_bitflags!{
     }
 }
 
+#[cfg(target_os = "android")]
```

Comment:
They are still available on other platforms, just defined as libc::c_int above ?

Maybe I should use cfg_if! to make it clearer.